test/run-test.ps1

```diff
@@ -85,10 +85,10 @@ Get-ChildItem -Path $repoRoot -Recurse -Filter Dockerfile |
             if ($platform -eq "linux") {
                 $selfContainedImage = "self-contained-build-${buildImage}"
                 $optionalRestoreParams = ""
-                if ($sdkTag -like "2.0-sdk") {
-                    # Temporary workaround until 2.0 packages are released on NuGet.org
-                    $optionalRestoreParams = "-s https://dotnet.myget.org/F/dotnet-core/api/v3/index.json -s https://api.nuget.org/v3/index.json"
-                }
+                # Enable and update as appropriate whenever pre-release packages are referenced prior to being available on NuGet.org.
+                # if ($sdkTag -like "<???>-sdk") {
+                #     $optionalRestoreParams = "-s https://dotnet.myget.org/F/dotnet-core/api/v3/index.json -s https://api.nuget.org/v3/index.json"
+                # }
```

Comment:
I'm not real fond of this pattern to begin with.  I see opportunity to clean some of this up once multi-stage builds are available.  I changed as suggested.